File: AndroidJetpackComposeSample/ffmpeg/src/main/java/com/nxg/ffmpeg/NativeLib.java
package com.nxg.ffmpeg;

public class NativeLib {

    // Used to load the 'ffmpeg' library on application startup.
    static {
        System.loadLibrary("ffmpeg");
    }

    /**
     * A native method that is implemented by the 'ffmpeg' native library,
     * which is packaged with this application.
     */
    //public native String stringFromJNI();
}

File: AndroidJetpackComposeSample/WebrtcMobile/src/main/java/com/nxg/webrtcmobile/NativeLib.kt
package com.nxg.webrtcmobile

class NativeLib {

    /**
     * A native method that is implemented by the 'webrtcmobile' native library,
     * which is packaged with this application.
     */
    external fun stringFromJNI(): String

    companion object {
        // Used to load the 'webrtcmobile' library on application startup.
        init {
            System.loadLibrary("webrtcmobile")
        }
    }
}